File: apps/problems/Anagram/boilerPlate-Full/function.js

  ##USER_CODE_HERE##
  
  const input = require('fs').readFileSync('/dev/stdin', 'utf8').trim().split('\n').join(' ').split(' ');
  const s1 = parseInt(input.shift());
  const s2 = parseInt(input.shift());
  const result = areAnagrams(s1, s2);
  console.log(result);
      

File: apps/problems/Anagram/boilerPlate-Full/function.rs

  use std::io::{self, BufRead};
  
  ##USER_CODE_HERE##
  
  fn main() {
    let stdin = io::stdin();
    let mut input = stdin.lock().lines().map(|l| l.unwrap());
    let s1: unknown = input.next().unwrap().parse().unwrap();
  let s2: unknown = input.next().unwrap().parse().unwrap();
    let result = areAnagrams(s1, s2);
    println!("{}", result);
  }
      